libs/server/Resp/RespCommand.cs

```diff
@@ -74,10 +74,10 @@ public enum RespCommand : byte
         ZREVRANGE,
         ZREVRANK,
         ZSCAN,
-        ZSCORE,
+        ZSCORE, // Note: Update OneIfRead if adding new read commands after this
```

Comment:
@kevin-montrose - I added these comments to the enum to make sure we don't later add entries here without adjusting the boundary conditions.